README.md

```diff
@@ -41,13 +41,26 @@ If the Multisite Privacy plugin or the More Privacy Option plugins are installed
 
 1. Copy the ed-lti folder to the plugins folder.
 **Note:** for instructions on how to install a plugin manually, please visit [WordPress for dummies](https://www.dummies.com/web-design-development/wordpress/templates-themes-plugins/how-to-install-wordpress-plugins-manually/)
-2. From **Network Admin > Plugins** select the plugin and click **Network Activate**.
-3. Go to **Network Admin > Settings > LTI Platforms** and enter a Name, Issuer (URL of connecting VLE, e.g. http://moodle.test) Client ID, Deployment ID, Authentication Request URL, Access Token URL, Public Keyset URL. These properties should be provided by the VLE. For example, on moodle, once you have created a tool, click on the 'View configuration details' icon to work out what values to enter for the platform.
+2. Either locally (before you push the plugin) or on the server, you will need to run composer to install the imsglobal libraries.
```

Comment:
can we change this to provide a bit more information for people that might not be familiar with Composer? Something like:

> ...you will need to run [Composer](https://getcomposer.org), a package manager for PHP, to install all dependency libraries for this plugin.

Reply:
Added - or something similar

Reply:
Cheers!